Bottom: 92f329dcd2844a03987a4aa171220e59004cab1c
Top:    d58dfe20a8c9ef24ce8601ebb799a233b57faa2a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 17:57:50 +0000

Refresh of move-some-fields-from-the-hid

---

diff --git a/src/hid.h b/src/hid.h
index b9c64ba..50921e9 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -256,7 +256,7 @@ typedef enum
     /* If set, this is the GUI HID.  Exactly one of these three flags
        must be set; setting "gui" lets the expose callback optimize and
        coordinate itself.  */
-    char _gui:1;
+    char gui:1;
 
     /* If set, this is the printer-class HID.  The common part of PCB
        may use this to do command-line printing, without having
diff --git a/src/main.c b/src/main.c
index 6e30752..58ba024 100644
--- a/src/main.c
+++ b/src/main.c
@@ -230,7 +230,7 @@ usage_hid (HID * h)
   int i, n_attributes = 0;
   UsageNotes *note;
 
-  if (h->_gui)
+  if (h->gui)
     {
       fprintf (stderr, "\n%s gui options:\n", h->name);
       attributes = h->get_export_options (&n_attributes);
@@ -263,7 +263,7 @@ usage (void)
 
   for (i = 0; hl[i]; i++)
     {
-      if (hl[i]->_gui)
+      if (hl[i]->gui)
 	n_gui++;
       if (hl[i]->printer)
 	n_printer++;
